plant-collect.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+//Create function to initiate connection with database and create query
+
+function connectDb()
+{
+    $db = new PDO (
+        'mysql:host=DB;dbname=example', 'root', 'password'
+    );
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function createQuery(){
+    $db = connectDb();
```

Comment:
I would prefer if this function had its database passed in from where it is called - in the index.php.